SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1110,14 +1100,6 @@ AActor* USpatialReceiver::CreateActor(UnrealMetadata* UnrealMetadataComp, SpawnD
 		return nullptr;
 	}
 
-	const bool bIsServer = NetDriver->IsServer();
-
-	// Initial Singleton Actor replication is handled with GlobalStateManager::LinkExistingSingletonActors
-	if (bIsServer && ActorClass->HasAnySpatialClassFlags(SPATIALCLASS_Singleton))
-	{
-		return FindSingletonActor(ActorClass);
-	}
-
```

Comment:
Instead, singletons now follow the StablyNamedActor logic from the `TryGetOrCreateActor` method above